packages/core/components/Puck/styles.module.css

```diff
@@ -119,13 +155,14 @@
   display: flex;
   flex-direction: column;
   grid-area: editor;
-  overflow-y: auto;
+  overflow: auto;
   position: relative;
 }
 
 .Puck-root {
   box-shadow: 0 0 0 calc(var(--puck-space-px) * 2) var(--puck-color-grey-10);
   margin: var(--puck-space-px);
+  min-width: 321px;
```

Comment:
I think the minimum should be bigger, to serve iPhone ~12 dimensions by default (488px or whatever it is). Makes a big difference to me when testing.

Reply:
Mostly agree, but will require some unpicking, so suggest we do in post.

Reply:
Spun out to #247